docs/index.rst

```diff
@@ -87,6 +87,7 @@ Documentation
 .. toctree::
    :maxdepth: 1
 
+   schema_registry_tutorial
```

Comment:
please move this lower down on the table of contents (shouldn't be the first thing on the nav) - maybe below `config`.

Reply:
@joel-hamill : http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/1284/5.0.0/schema-registry/docs/schema_registry_tutorial.html